src/main/java/com/avaya/letscode/Cart.java

```diff
@@ -33,13 +33,19 @@ public int getQuantity() {
 
     public double getCartValue() {
```

Comment:
Can we call this simply getValue() as we are already inside the Cart object. This will be on consistent lines with getQuantity().